modules/new_serverpod_auth/serverpod_auth_backwards_compatibility/serverpod_auth_backwards_compatibility_server/test/integration/backwards_compatibility_test.dart

```diff
@@ -0,0 +1,32 @@
+import 'package:serverpod/serverpod.dart';
+import 'package:serverpod_auth_backwards_compatibility_server/serverpod_auth_backwards_compatibility_server.dart';
+import 'package:test/test.dart';
+
+import './test_tools/serverpod_test_tools.dart';
+
+void main() {
+  withServerpod(
+    'Given no legacy passwords,',
+    (final sessionBuilder, final endpoints) {
+      late Session session;
+
+      setUp(() {
+        session = sessionBuilder.build();
+      });
+
+      test(
```

Comment:
the tests for actual "full imports" (migration + backwards compatibility) are below, as that can call into the real code and I don't have to mock the migrated stuff (which is not easy to verify then).